File: Molinos.Scato.Dominio/Entidades/PlanoDeCargaBodegaDestino.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Molinos.Scato.Dominio.Entidades
{
    public class PlanoDeCargaBodegaDestino : IIdentificable
    {
        public virtual int Id { get; set; }
        public virtual PlanoDeCargaBodega PlanoDeCargaBodega { get; set; }
        public virtual Destino Destino { get; set; }
        public virtual decimal Cantidad { get; set; }
    }
}

File: Molinos.Scato.WebPuerto/src/app/shared/modelos/plano-de-carga-bodega-destino.ts
import { Destino } from "./destino";
import { Exportador } from "./exportador";

export class PlanoDeCargaBodegaDestino {
  id: number;
  destino: Destino;
  cantidad: number;
  exportador?: Exportador;

  constructor(id = 0, destino: Destino, cantidad: number, exportador?: Exportador) {
    this.id = id;
    this.destino = destino;
    this.cantidad = cantidad;
    this.exportador = exportador;
  }
}
